src/Caliburn.Micro.Platform/Caliburn.Micro.Platform.csproj

```diff
@@ -76,4 +84,9 @@
     <Compile Include="ViewModelLocator.cs" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Update="Microsoft.SourceLink.GitHub" Version="1.0.0" />
+    <PackageReference Update="Nerdbank.GitVersioning" Version="3.3.37" />
```

Comment:
You'll want to update the versions in https://github.com/Caliburn-Micro/Caliburn.Micro/blob/master/src/Directory.Build.props which will affect all projects rather that just individual projects.

Reply:
Updated

Reply:
Should be able to drop the references from the projects as well.

Reply:
done